<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-L5HGJIRM" ignored="0" name="FREE CRM Test" tests="1" failures="1" timestamp="17 Aug 2021 12:18:02 GMT" time="10.977" errors="0">
  <testcase name="feature" time="10.583" classname="MyRunner.TestRunner">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: expected [#1 Free CRM customer relationship management software cloud] but found [Free CRM in the cloud software boosts sales]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [#1 Free CRM customer relationship management software cloud] but found [Free CRM in the cloud software boosts sales]
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at MyRunner.TestRunner.feature(TestRunner.java:36)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: expected [#1 Free CRM customer relationship management software cloud] but found [Free CRM in the cloud software boosts sales]
at stepDefinitions.LoginStepDefinition.title_of_login_page_is_free_CRM(LoginStepDefinition.java:32)
at âœ½.When title of login page is Free CRM(login.feature:6)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- FREE CRM Test -->
